### ISSUE TITLE
Assignment 17

### ISSUE THREAD

**Issue by FSGabrsek** [FSGabrsek/AE-I_Intro-Data-Science#23]
### Decision tree regression

- [x] Split the penguins dataset into a training and test set
- [x] Fit a DecisionTreeClassifier with a valid training data
- [x] Document the RMSE of the tree for the chosen set size and depth
